tests/e2e/util/statefulset/statefulset.go

```diff
@@ -62,3 +67,14 @@ func IsAllDesiredPodsRunningAndReady(c kubernetes.Interface, sts *appsv1.Statefu
 	klog.Infof("desired pods of sts %s/%s are running and ready (%v)", sts.Namespace, sts.Name, actualPodOrdinals.List())
 	return true
 }
+
+// GetPodList gets the current Pods in ss.
+// e2esset.GetPodList(c, sts) would panic while we want return error if something wrong
+func getPodList(c kubernetes.Interface, ss *appsv1.StatefulSet) (*corev1.PodList, error) {
```

Comment:
The original `e2esset.GetPodList(c, sts)` would panic if any error happened, we replace it and the newer function would return an error if error happened.